### ISSUE TITLE
res.json(null) error

### ISSUE THREAD

**Issue by willywotz** [bradoyler/route-cache#34]
simple my code
```
const express = require('express') // 4.x
const microcache = require('route-cache')

const app = express()
app.use(microcache.cacheSeconds(1))

app.all('*', (req, res) => res.json(null))

app.listen(8080, _ => console.log('started listening'))
```

on call i get
```
TypeError: Cannot read property 'length' of null
```

**Comment by amangqq** [comment]
解决了吗

**Comment by bradoyler** [comment]
error is from this line: https://github.com/bradoyler/route-cache/blob/master/index.js#L77

**Comment by bradoyler** [comment]
fixed via #42
